freqtrade/configuration/arguments.py

```diff
@@ -40,6 +40,8 @@
 ARGS_PLOT_PROFIT = (ARGS_COMMON + ARGS_STRATEGY +
                     ["pairs", "timerange", "export", "exportfilename", "db_url", "trade_source"])
 
+NO_CONF_REQURIED = ["start_download_data"]
```

Comment:
It may be a list of RunMode's (RunMode.DownloadData, RunMode.ListPairs etc.) corresponding to subcommands and, therefore, to the start_* functions. This just a note, I'm not sure if this is better than this list...

Reply:
Yes i agree in theory - but then we  still need a mapping of runmode -> start* function - since the only thing available at the point of the check is the function name that's called (`func.__name__`).

Remember that this is before we have config loaded - so not much usefull will be available...